src/util/Makefile

```diff
@@ -28,6 +28,7 @@ SRC = allocate_objects.cpp \
       get_module.cpp \
       identifier.cpp \
       ieee_float.cpp \
+      interval_union.cpp \
```

Comment:
From the commit message it sounds like this will be removed from test-gen, in a bump presumably?

Reply:
https://github.com/diffblue/test-gen/pull/3970